scipy/stats/stats.py

```diff
@@ -237,6 +237,10 @@ def _chk2_asarray(a, b, axis):
     return a, b, outaxis
 
 
+def _contains_nan(a):
+    return np.isnan(np.sum(a))
```

Comment:
It's worth a comment here as to why not ` isnan(a).any()` (what you wrote in the GH comment)